git_log.go

```diff
@@ -0,0 +1,34 @@
+// Copyright ©2022 Dan Kortschak. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+
+	"github.com/kortschak/hunspell"
+	"golang.org/x/sys/execabs"
+)
+
+// readGitLog adds author names and email addresses from git log.
+func readGitLog(spelling *hunspell.Spell) {
+	cmd := execabs.Command("git", "log", "--format=%an %ae")
```

Comment:
This is unfortunate, but `git shortlog -sn` will not return the bytes that we need, so we need to process the full history here.